ResearchKit/Common/ORKNavigationContainerView.m

```diff
@@ -254,12 +254,17 @@ - (void)updateConstraints {
                                                                   attribute:NSLayoutAttributeNotAnAttribute
                                                                  multiplier:1 constant:0]];
     } else if (_neverHasContinueButton) {
-        [_localConstraints addObject:[NSLayoutConstraint constraintWithItem:_continueButton
-                                                                  attribute:NSLayoutAttributeHeight
-                                                                  relatedBy:NSLayoutRelationEqual
-                                                                     toItem:nil
-                                                                  attribute:NSLayoutAttributeNotAnAttribute
-                                                                 multiplier:1 constant:0]];
+        NSLayoutConstraint *heightConstraint = [NSLayoutConstraint constraintWithItem:_continueButton
+                                                                            attribute:NSLayoutAttributeHeight
+                                                                            relatedBy:NSLayoutRelationEqual
+                                                                               toItem:nil
+                                                                            attribute:NSLayoutAttributeNotAnAttribute
+                                                                           multiplier:1
+                                                                             constant:0];
+        
+        // This covers the case in which there is no continue button but there is a skip button.
```

Comment:
Great, this is very clear. One little request, can you mention HanoiTask in this line?